src/segments/Header.tsx

```diff
@@ -12,7 +12,7 @@ import styled from 'styled-components';
 
 // Internal Components
 import { Alert } from '@site/src/components/Alert';
```

Comment:
See how docs convert icon into pink via filters, use that instead of image change